.github/workflows/lint.yml

```
@@ -0,0 +1,16 @@
+name: lint
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+jobs:
+  run_lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
```

Rather than explicitly stating our Node version here, and risking drift from our other tooling, what do you think about using a `.node-version` file? And then:

```suggestion
          node-version-file: ".node-version"
```

---

Oh wow this is a neat trick!  Side note: fully agree with use of `.node-version` since that becomes an artifact that others can use (e.g. they can symlink it to an .nvmrc).